package.json

```diff
@@ -1,18 +1,36 @@
 {
-	"private": true,
-	"name": "o-date",
+	"name": "@financial-times/o-date",
+	"version": "0.0.0",
+	"type": "module",
+	"browser": "main.js",
+	"description": "JavaScript utilities for formatting and updating dates in the FT style. Also useful for formatting dates relative to the current time.",
+	"keywords": [
+		"datetime",
+		"time",
+		"timestamp"
+	],
+	"homepage": "https://registry.origami.ft.com/components/o-date",
+	"bugs": {
+		"url": "https://registry.origami.ft.com/components/o-date"
+	},
+	"license": "MIT",
+	"engines": {
+		"npm": "^7"
+	},
 	"devDependencies": {
 		"eslint": "^7.8.1",
 		"eslint-config-origami-component": "^2.1.0",
-		"origami-ci-tools": "^2.0.2",
 		"remark": "^13.0.0",
 		"remark-lint": "^8.0.0",
-		"remark-preset-lint-origami-component": "^1.1.3",
+		"remark-preset-lint-origami-component": "prerelease",
 		"stylelint": "^13.7.0",
 		"stylelint-config-origami-component": "^1.0.4"
 	},
 	"scripts": {
 		"start": "npx serve ./demos/local",
-		"build": "npm_config_yes=true npx \"origami-build-tools@^10\" install && npm_config_yes=true npx \"origami-build-tools@^10\" demo"
+		"build": "npm_config_yes=true npx \"origami-build-tools@prerelease\" install && npm_config_yes=true npx \"origami-build-tools@prerelease\" demo"
```

Comment:
how come o-date is special? shall we remove these scripts?

Reply:
What do you mean by special? This is in most of our components to get branch previews working :-D

Reply:
Oh! Got it